server/errors/ApplicationErrors.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Can you provide some more context on this new addition?

Reply:
I think you are right and I should add it as a new PR. I added it initially because ESLint requires Error objects to be the only item thrown.  See: https://github.com/donthedeveloper/site-builder/pull/103#discussion_r261873090

I figured this was the cleanest way to create an Error object the way Ben wanted and leave us the ability to be able to create more custom errors when needed. For example, https://github.com/donthedeveloper/site-builder/pull/117#discussion_r263981329.

I am not sure if this is necessary. What are your thoughts?

Reply:
Per our conversation, this has been removed.